File: src/main/java/com/catan/catanbackend/model/dto/PlayerTradeDto.java
package com.catan.catanbackend.model.dto;

import com.catan.catanbackend.model.ResourceGroup;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class PlayerTradeDto {
    private Long sessionId;
    private String fromUser;
    private String toUser;
    private ResourceGroup offered;
    private ResourceGroup requested;
}

File: Assets/Scripts/TradingReasources/Models/PlayerTradeDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Assets.Scripts.TradingReasources.Models
{

    [Serializable]
    public class PlayerTradeDto
    {
        public long sessionId;
        public string fromUser;
        public string toUser;
        public ResourceGroup offered;
        public ResourceGroup requested;
    }
}
